woodstock/voting/static/js/events.js

```diff
@@ -0,0 +1,19 @@
+$(document).ready(function () {
+    'use strict';
+
+    $('#blind-info').tooltip();
+
+    function generate_click() {
+        var events = [];
+        $('input:checkbox:checked').each(function(){
+            events.push($(this).val());
+        });
+        var blind = $('input:radio:checked').val();
+
+        $('#events-submit').attr('href', '/dashboard/?events=' + events + '&blind=' + blind);
+    }
+
+    $('input').change(function() {
+        generate_click();
```

Comment:
No need to define such a specific method. Move ``generate_click`` definition here.